app/controllers/public_pages_controller.rb

```diff
@@ -4,13 +4,9 @@ class PublicPagesController < ApplicationController
   # GET template_index
   # -----------------------------------------------------
   def template_index
-    template_ids = Template.where(org_id: Org.funder.pluck(:id), visibility: Template.visibilities[:publicly_visible]).valid.pluck(:dmptemplate_id).uniq << Template.where(is_default: true, published: true).pluck(:dmptemplate_id)
-    @templates = []
-    template_ids.flatten.each do |tid|
-      t = Template.live(tid)
-      @templates << t unless t.nil?
-    end
-    @templates = @templates.sort{|x,y| x.title <=> y.title } if @templates.count > 1
+    template_ids = Template.families(Org.funder.pluck(:id)).publicly_visible.pluck(:dmptemplate_id) <<
+    Template.where(is_default: true).valid.published.pluck(:dmptemplate_id)
```

Comment:
I know we are doing this other places already. It would be good to refactor this and the paginable publicly_visible so that we are DRY. Can wait until after MVP though when we do general cleanup